Documentation/operations/performance/tuning.rst

```diff
@@ -339,10 +339,111 @@ performance. The overhead of Hubble is somewhere between 1-15% depending
+
+By default the Hubble event queue size is ``#CPU * 1024``, or ``16384`` if your nodes have
+more than 16 CPU cores. If you encounter event bursts that result in dropped events,
+increasing this queue size might help. We recommend gradually doubling the queue length
+until the drops disappear. If you don't see any improvements after increasing the queue
+length to 128k, further increasing the event queue size is unlikely to help.
+
+Be aware that increasing the Hubble event queue size will result in increased memory
+usage. Depending on your traffic pattern, increasing the queue size by ``10,000`` may
+increase the memory usage by up to five Megabytes.
+
+.. tabs::
+
+    .. group-tab:: Cilium CLI
+
+       .. parsed-literal::
+
+           cilium install |CHART_VERSION| \\
+             --set hubble.eventQueueSize=32768
+
+    .. group-tab:: Helm
+
+       .. parsed-literal::
+
+           helm install cilium |CHART_RELEASE| \\
+             --namespace kube-system \\
+             --set hubble.eventQueueSize=32768
+
+    .. group-tab:: Per-Node
+
+      If only certain nodes are effected you may also set the queue length on a per-node
+      basis using a :ref:`CiliumNodeConfig object <per-node-configuration>`.
+
+      ::
+
+          apiVersion: cilium.io/v2
+          kind: CiliumNodeConfig
+          metadata:
+            namespace: kube-system
+            name: set-hubble-event-queue
+          spec:
+            nodeSelector:
+              matchLabels:
+                # Update selector to match your nodes
+                io.cilium.update-hubble-event-queue: "true"
+            defaults:
+              hubble-event-queue-size: "32768"
+
+Increasing the Hubble event queue size can't mitigate a consistently high rate of events
+being emitted by Cilium datapath and it does not reduce CPU utilization. For this you
+should consider increasing the aggregation interval or rate limiting events.
+
+Increase Aggregation Interval
+-----------------------------
+
+By default Cilium generates a tracing event on every new connection, any time a packet
+contains TCP flags that have not been previously seen for the packet direction, and on
+average once per ``monitor-aggregation-interval``, which defaults to 5 seconds.
+
+Depending on your network traffic patterns, the re-emitting of trace events per
+aggregation interval can make up a large part of the total events. Increasing the
+aggregation interval may decrease CPU utilization and can prevent lost events.
+
+The following will set the aggregation interval to 10 seconds.
+
+.. tabs::
+    .. group-tab:: Cilium CLI
+
+       .. parsed-literal::
+
+           cilium install |CHART_VERSION| \\
+             --set bpf.events.monitorInterval="10s"
+
+    .. group-tab:: Helm
+
+       .. parsed-literal::
+
+           helm install cilium |CHART_RELEASE| \\
+             --namespace kube-system \\
+             --set bpf.events.monitorInterval="10s"
```

Comment:
"bpf.**events**.monitorInterval"
and
"bpf.**events**.monitorInterval"
seems not existing. Is it a regression? I think the key `events` is obsolete and the interval can be controlled directly via `bpf.monitorInterval`: https://github.com/search?q=repo%3Acilium%2Fcilium%20%22bpf.monitorInterval%22&type=code